depends: fallback server missing Qt downloads

Looks like `https://code.qt.io` is down / having issues. None of the files needed from that site are available from the fallback server:
```bash
Checksum missing or mismatched for qt source. Forcing re-download.
Fetching qtbase-everywhere-src-6.7.3.tar.xz from https://download.qt.io/archive/qt/6.7/6.7.3/submodules
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   281  100   281    0     0    306      0 --:--:-- --:--:-- --:--:--   306
100 47.1M  100 47.1M    0     0  4687k      0  0:00:10  0:00:10 --:--:-- 7798k
/bitcoin/depends/work/download/qt-6.7.3/qtbase-everywhere-src-6.7.3.tar.xz.temp: OK
Fetching qttranslations-everywhere-src-6.7.3.tar.xz from https://download.qt.io/archive/qt/6.7/6.7.3/submodules
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   289  100   289    0     0    284      0  0:00:01  0:00:01 --:--:--   285
100 1511k  100 1511k    0     0   622k      0  0:00:02  0:00:02 --:--:-- 1430k
/bitcoin/depends/work/download/qt-6.7.3/qttranslations-everywhere-src-6.7.3.tar.xz.temp: OK
Fetching qttools-everywhere-src-6.7.3.tar.xz from https://download.qt.io/archive/qt/6.7/6.7.3/submodules
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   282  100   282    0     0    336      0 --:--:-- --:--:-- --:--:--   336
100 8811k  100 8811k    0     0  2715k      0  0:00:03  0:00:03 --:--:-- 3864k
/bitcoin/depends/work/download/qt-6.7.3/qttools-everywhere-src-6.7.3.tar.xz.temp: OK
Fetching CMakeLists.txt?h=6.7.3 from https://code.qt.io/cgit/qt/qt5.git/plain
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:30 --:--:--     0
curl: (28) Failed to connect to code.qt.io port 443 after 30002 ms: Timeout was reached
Warning: Problem : timeout. Will retry in 1 seconds. 3 retries left.
  0     0    0     0    0     0      0      0 --:--:--  0:00:30 --:--:--     0
curl: (28) Failed to connect to code.qt.io port 443 after 30006 ms: Timeout was reached
Warning: Problem : timeout. Will retry in 2 seconds. 2 retries left.
  0     0    0     0    0     0      0      0 --:--:--  0:00:30 --:--:--     0
curl: (28) Failed to connect to code.qt.io port 443 after 30005 ms: Timeout was reached
Warning: Problem : timeout. Will retry in 4 seconds. 1 retries left.
  0     0    0     0    0     0      0      0 --:--:--  0:00:30 --:--:--     0
curl: (28) Failed to connect to code.qt.io port 443 after 30004 ms: Timeout was reached
Fetching CMakeLists.txt?h=6.7.3 from https://bitcoincore.org/depends-sources
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0   146    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
make[1]: *** [funcs.mk:346: /bitcoin/depends/sources/download-stamps/.stamp_fetched-qt-qtbase-everywhere-src-6.7.3.tar.xz.hash] Error 22
make[1]: Leaving directory '/bitcoin/depends'
make: *** [Makefile:276: download-linux] Error 2
make: Leaving directory '/bitcoin/depends'
```

--- a/depends/packages/qt_details.mk
+++ b/depends/packages/qt_details.mk
@@ -13,14 +13,14 @@ qt_details_qttools_sha256_hash := f03bb7df619cd9ac9dba110e30b7bcab5dd88eb8bdc9cc
 
 qt_details_patches_path := $(PATCHES_PATH)/qt
 
-qt_details_top_download_path := https://code.qt.io/cgit/qt/qt5.git/plain
+qt_details_top_download_path := https://raw.githubusercontent.com/qt/qt5/refs/heads/$(qt_details_version)
 qt_details_top_cmakelists_file_name := CMakeLists.txt
-qt_details_top_cmakelists_download_file := $(qt_details_top_cmakelists_file_name)?h=$(qt_details_version)
+qt_details_top_cmakelists_download_file := $(qt_details_top_cmakelists_file_name)
 qt_details_top_cmakelists_sha256_hash := 9fb720a633c0c0a21c31fe62a34bf617726fed72480d4064f29ca5d6973d513f
 qt_details_top_cmake_download_path := $(qt_details_top_download_path)/cmake
 qt_details_top_cmake_ecmoptionaladdsubdirectory_file_name := ECMOptionalAddSubdirectory.cmake
-qt_details_top_cmake_ecmoptionaladdsubdirectory_download_file := $(qt_details_top_cmake_ecmoptionaladdsubdirectory_file_name)?h=$(qt_details_version)
+qt_details_top_cmake_ecmoptionaladdsubdirectory_download_file := $(qt_details_top_cmake_ecmoptionaladdsubdirectory_file_name)
 qt_details_top_cmake_ecmoptionaladdsubdirectory_sha256_hash := 97ee8bbfcb0a4bdcc6c1af77e467a1da0c5b386c42be2aa97d840247af5f6f70
 qt_details_top_cmake_qttoplevelhelpers_file_name := QtTopLevelHelpers.cmake
-qt_details_top_cmake_qttoplevelhelpers_download_file := $(qt_details_top_cmake_qttoplevelhelpers_file_name)?h=$(qt_details_version)
+qt_details_top_cmake_qttoplevelhelpers_download_file := $(qt_details_top_cmake_qttoplevelhelpers_file_name)
 qt_details_top_cmake_qttoplevelhelpers_sha256_hash := 5ac2a7159ee27b5b86d26ecff44922e7b8f319aa847b7b5766dc17932fd4a294

